ibu-imager/ostreeclient/rpmostreeclient.go

```diff
@@ -112,3 +119,37 @@ func (c *Client) QueryStatus() (*Status, error) {
 
 	return &q, nil
 }
+
+// IsStaterootBooted returns whether the specified stateroot is booted
+func (c *Client) IsStaterootBooted(stateroot string) (bool, error) {
+	status, err := c.QueryStatus()
+	if err != nil {
+		return false, err
+	}
+
+	for _, deployment := range status.Deployments {
+		if deployment.Booted && deployment.OSName == stateroot {
+			return true, nil
+		}
+	}
+	return false, nil
+}
+
+type FakeClient struct {
```

Comment:
Can we move to use mocks please? Ostree client should implement interface and interface can be mocked easily with https://github.com/golang/mock. It will be much more generic than creating fake struct per each object.

Reply:
As here https://github.com/leo8a/ibu-imager/pull/12/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R51

Reply:
Agreed..also If possible consider updating the mod file to https://github.com/uber-go/mock since what we have currently is archived